src/db/sqlite/lrsql/sqlite/sql/delete.sql

```diff
@@ -59,3 +59,53 @@ DELETE FROM credential_to_scope
 WHERE api_key = :api-key
 AND secret_key = :secret-key
 AND scope = :scope
+
+-- :name delete-actor-st2actor!
+-- :command :execute
+-- :result :affected
+DELETE FROM statement_to_actor where actor_ifi = :actor-ifi
+
+-- :name delete-actor-st2st!
+-- :command :execute
+-- :result :affected
+DELETE FROM statement_to_statement WHERE descendant_id in (:del-statement-ids)
```

Comment:
i do not think that IN clauses work with just a :del-statement-ids. We need to use a :v*: param to populate a list like that.